recipes/libnice/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+import os
+from conan import ConanFile
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.files import copy, get, rmdir, rename, chdir, replace_in_file
+from conan.tools.layout import basic_layout
+from conan.tools.gnu import PkgConfigDeps
+from conan.errors import ConanInvalidConfiguration
+
+
+class LibniceConan(ConanFile):
+    name = "libnice"
+    version = "0.1.19"
+    homepage = "https://libnice.freedesktop.org/"
+    license = "MPL-1.1 AND LGPL-2.1-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "a GLib ICE implementation"
+    topics = ("ice", "stun", "turn")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "crypto_library": ["openssl", "win32"],
+        "with_gstreamer": [True, False],
+        "with_gtk_doc": [True, False],
+        "with_introspection": [True, False]}
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_gstreamer": False,
+        "with_gtk_doc": False,
+        "with_introspection": False}
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            self.options.crypto_library = "win32"
+        else:
+            self.options.crypto_library = "openssl"
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
```

Comment:
```suggestion
           self.options.rm_safe("fPIC")
       self.settings.rm_safe("compiler.libcxx")
       self.settings.rm_safe("compiler.cppstd")
```

https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#settings